[2023-06-09 15:27:45,123] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_18.csv manual__2023-06-09T15:20:24.730543+00:00 [queued]>
[2023-06-09 15:27:45,181] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_18.csv manual__2023-06-09T15:20:24.730543+00:00 [queued]>
[2023-06-09 15:27:45,182] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 15:27:45,183] {taskinstance.py:1242} INFO - Starting attempt 3 of 3
[2023-06-09 15:27:45,184] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 15:27:45,234] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): load_sftp_datadata_18.csv> on 2023-06-09 15:20:24.730543+00:00
[2023-06-09 15:27:45,243] {standard_task_runner.py:52} INFO - Started process 4227 to run task
[2023-06-09 15:27:45,250] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'sftp_raw_data_full_load_pipeline', 'load_sftp_datadata_18.csv', 'manual__2023-06-09T15:20:24.730543+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/sftp_raw_data_full_load_pipeline.py', '--cfg-path', '/tmp/tmpr4gxecse', '--error-file', '/tmp/tmpl80dc4lk']
[2023-06-09 15:27:45,258] {standard_task_runner.py:77} INFO - Job 92: Subtask load_sftp_datadata_18.csv
[2023-06-09 15:27:45,354] {logging_mixin.py:109} INFO - Running <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_18.csv manual__2023-06-09T15:20:24.730543+00:00 [running]> on host 8cb5811d1998
[2023-06-09 15:27:45,472] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=Hassan@gmail.com
AIRFLOW_CTX_DAG_OWNER=Hassan
AIRFLOW_CTX_DAG_ID=sftp_raw_data_full_load_pipeline
AIRFLOW_CTX_TASK_ID=load_sftp_datadata_18.csv
AIRFLOW_CTX_EXECUTION_DATE=2023-06-09T15:20:24.730543+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-09T15:20:24.730543+00:00
[2023-06-09 15:27:45,494] {logging_mixin.py:109} INFO - error while inserting to table: Invalid file path or buffer object type: <class 'NoneType'>
[2023-06-09 15:27:45,543] {local_task_job.py:154} INFO - Task exited with return code 1
[2023-06-09 15:27:45,603] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=sftp_raw_data_full_load_pipeline, task_id=load_sftp_datadata_18.csv, execution_date=20230609T152024, start_date=20230609T152745, end_date=20230609T152745
[2023-06-09 15:27:45,710] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
